site/docs/patterns/index.mdx

```diff
@@ -0,0 +1,524 @@
+  same form field). In this case, use top place labels.
+</ExampleContainer>
+
+## Label placement
+
+Forms either support a top or left label placement.
+
+### Top placement
+
+The default label placement for Form Fields. Top placement offers:
+
+- Most scannable solution, allowing users to traverse the left edge of the form through labels, fields and, finally, onward actions.
+- Fastest completion times.
+- Best label placement across varying devices and viewport sizes.
+- Most accessible/compatible with browser zoom.
+
+<Diagram
+  border
+  src="/img/patterns/forms/placement_1.png"
+  alt="Example showing top placement"
+/>
+
+### Left placement
+
+- A label placed to the left of the form control can be aligned left or right.
+- Often used in very dense, "compact" forms, where quantity of visible fields is more important than completion time and viewport size.
+
+<Diagrams>
+  <Diagram
+    border
+    src="/img/patterns/forms/placement_2.png"
+    alt="Example showing right aligned labels"
+    contentPosition="top"
+  >
+    #### Right aligned
+    - Default aligned for left placement.
+    - Maintains proximity of labels to fields.
+    - Often used for dense editable forms, such as tickets.
+  </Diagram>
+  <Diagram
+    border
+    src="/img/patterns/forms/placement_3.png"
+    alt="Example showing left aligned labels"
+    contentPosition="top"
+  >
+    #### Left aligned
+    - Commonly when users need to review, or take care entering form information.
+    - Generally slower completion time (due to variable label proximity to field).
+    - Use cases include form summaries or to collect sensitive information.
+  </Diagram>
+</Diagrams>
+
+<ExampleContainer type="positive">
+  - Ensure both top and left placed labels align to the same layout grid configuration.
+  - All left placement labels in the form should have the same width (determined by the field with the longest label).
+  - Avoid left placement when the form is displayed in different languages, or on mobile/smaller viewports.
+</ExampleContainer>
+
+<ExampleContainer type="negative">
+  - We don't recommend mixing label placements across the same form.
+</ExampleContainer>
+
+## Headings
+
+Headings create additional visual separation (after choosing field and background fill colour) and group related fields, to help users complete forms:
+
+- Space can be added above each heading for additional separation in the form, if space allows.
+- If your heading is followed by a description, spacing is used to reinforce the relationship between heading and description.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/forms/heading_1.png", alt: "" },
+    { src: "/img/patterns/forms/heading_2.png", alt: "" },
+  ]}
+  label="Show spacing"
+/>
+
+<ExampleContainer type="negative">
+  Space above headings is optional. It shouldn’t be added when:
+  - The heading is at the beginning of the form.
+  - The heading follows a separator. - Using a “Compact form” layout.
```

Comment:
```suggestion
  - The heading follows a separator. 
  - Using a “Compact form” layout.
```

Reply:
Thanks, there's a weird formatting issue that when prettier runs it messes up all the mdx